content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -110,6 +111,29 @@ either `Delete` or `Retain`. If no `reclaimPolicy` is specified when a
 Persistent Volumes that are created manually and managed via a storage class will have
 whatever reclaim policy they were assigned at creation.
 
+### Allow Volume Expansion
+
+{{< feature-state for_k8s_version="v1.11" state="beta" >}}
+
+Persistent Volumes can be configured to be expandable. This feature when set to `true`, 
+allows the users to resize the volume by editing the corresponding PVC object. 
+
+The following types of volumes support volume expansion, when the underlying
+Storage Class has the field `allowVolumeExpansion` set to true.
+
+* gcePersistentDisk
+* awsElasticBlockStore
+* Cinder
+* glusterfs
+* rbd
+* Azure File
+* Azure Disk
+* Portworx
+* FlexVolumes
+* CSI {{< feature-state for_k8s_version="v1.14" state="alpha" >}}
+
+Note: This feature cannot be used to shrink volumes.
```

Comment:
Follow the [style guide conventions](https://kubernetes.io/docs/contribute/style/style-guide/#note) for notes.

Reply:
Thank you, @zacharysarah for the notes and pointer. I will take care of it today.

Reply:
Hi @zacharysarah,I have added appropriate note tag, also squashed all the commits related to this issue into one.